README.md

```diff
@@ -117,6 +117,12 @@ disabled_rules=import-ordering
 disabled_rules=indent
 ```
 
+## Online demo
+In case you would like to see how `ktlint` works and you are not able (or do not want) to install it -
```

Comment:
I'm fine with the link. Can you please change the language to:

```
## Online demo
You can try `ktlint` online [here](https://ktlint-demo.herokuapp.com/) using the standard or a custom ruleset. To contribute, please visit the [GitHub repository](https://github.com/akuleshov7/diKTat-demo).
```

Reply:
updated the text

Reply:
LGTM